teams/tidb/membership.json

```diff
@@ -0,0 +1,79 @@
+{
+    "_comment": [
+        "Items of lists in this file are in alphabetical order. Please keep it on modification."
+    ],
+    
+    "maintainers": [
+        "XuHuaiyu",
+        "breeswish",
+        "c4pt0r",
+        "coocood",
+        "disksing",
+        "kennytm",
+        "ngaut",
+        "qiuyesuifeng",
+        "qw4990",
+        "shenli",
+        "siddontang",
+        "tangenta",
+        "tiancaiamao",
+        "winkyao",
+        "winoros",
+        "zimulala",
+        "zz-jason"
+    ],
+
+    "committers": [
+        "AilinKid",
+        "Deardrops",
+        "MyonKeminta",
+        "Reminiscent",
+        "SunRunAway",
+        "Yisaer",
+        "andylokandy",
+        "b41sh",
+        "bb7133",
+        "cfzjywxk",
+        "crazycs520",
+        "djshow832",
+        "eurekaka",
+        "francis0407",
+        "fzhedu",
+        "hanfei1991",
+        "ichn-hu",
+        "imtbkcat",
+        "jackysp",
+        "lysu",
+        "lzmhhh123",
+        "mmyj",
+        "morgo",
+        "nrc",
+        "rebelice",
+        "sticnarf",
+        "wjhuang2016",
+        "wshwsh12",
+        "xhebox",
+        "xiongjiwei",
+        "youjiali1995"
+    ],
+
+    "reviewers": [
+        "JmPotato",
```

Comment:
@breeswish I think there are several active contributor works on telemetry component for tidb. Are they deserved a reviewer or committer seat? It seems previously we don't have a proper SIG to accept them as sig member.

Reply:
@breeswish @kaixu120811 I'll close this conversation if no further input today.